l2l/tests/test_sa_optimizer.py

```diff
@@ -1,69 +1,31 @@
 import unittest
 
+from .test_optimizer import OptimizerTestCase
```

Comment:
Can you refactor this to an absoulte path, i.e. `from l2l.tests.test_optimizer import OptimzerTestCase`

Reply:
Yes sorry for that